WebContent/WOH-addRestaurant.jsp

```diff
@@ -31,13 +31,13 @@
 	<div class ="container">
 		<div class="row">
 			<div class="col-sm-10 col-sm-offset-1">
-				<form action = "" method = "POST" class="form-horizontal WOH-addRestaurant-form">
+				<form action = "Insert" method = "POST" class="form-horizontal WOH-addRestaurant-form">
 										
 					<!-- Restaurant Name Textbox -->
 						<div class="form-group">
 						<label class="control-label col-sm-2" for="WOH-addRestaurants-restaurantNameInput">Restaurant Name</label>
 							<div class="col-sm-10">
-	         					<input type = "text" class="form-control WOH-addRestaurant-restaurantNameInput" name = "name">
+	         					<input type = "text" class="form-control WOH-addRestaurants-restaurantNameInput" name = "name">
 	         				</div>
 	         			</div>
 	         			
```

Comment:
Change the price input into a drop down with options: $,$$,$$$,$$$$.  within formatData(), use a switch case to change these String values into their corresponding int values.